src/sentry/integrations/client.py

```diff
@@ -0,0 +1,236 @@
+from __future__ import absolute_import
+
+import logging
+import json
+import requests
+
+from BeautifulSoup import BeautifulStoneSoup
+from cached_property import cached_property
+from django.utils.datastructures import SortedDict
```

Comment:
I know kinda not related, but let's change this to just using an OrderedDict instead of this Django shim: https://code.djangoproject.com/wiki/SortedDict